README.md

```diff
@@ -41,18 +41,22 @@ or numerical analysis package.
 ### New Features in 3.1
 
 The 3.1 release of Imath introduces optimized half-to-float and
-float-to-half conversion, using Intel intrinsics if available, or
-otherwise using an optimized bit-manipulation algorithm that does not
-require lookup tables. Performance of both options is generally
+float-to-half conversion, using the F16C SSE extension if available,
+or otherwise using an optimized bit-manipulation algorithm that does
+not require lookup tables. Performance of both options is generally
 significantly faster than the lookup-table based conversions that
```

Comment:
To be clear, this is 100% true for the float->half using eLut. However, for the half->float, the table is still generally faster than the bit manipulation on x86 hardware, which is why it is still the default. The optimized code is provided for scenarios that do not want to link in a table of that size to their application, or for hardware where the table is slower.